views/index.erb

```diff
@@ -1,8 +1,31 @@
 <h1><%= @title %></h1>
 
 Output generated: <%= Time.now %>
+
 <pre>
 $ <%= @command %>
-
-<%= @output %>
 </pre>
+
+<table class="table table-bordered">
+  <tr>
+    <th>USER</th>
+    <th>PID</th>
+    <th>%CPU</th>
+    <th>%MEM</th>
+    <th>VSZ</th>
+    <th>RSS</th>
+    <th>TTY</th>
+    <th>STAT</th>
+    <th>START</th>
+    <th>TIME</th>
+    <th>COMMAND</th>
+  </tr>
+  <% @output.each do |row| %>
+  <tr>
+    <% row.each do |value| %>
+      <td><%= value %></td>
+    <% end %>
+  </tr>
```

Comment:
Actually yeah, it works well. Add:

```ruby
AppProcess = Struct.new(:user, :pid, :pct_cpu, :pct_mem, :vsz, :rss, :tty, :stat, :start, :time, :command)
```

Struct will accept each argument as a string (not keyword) so your array values match up perfectly. So you can just map the results of the parse command to AppProcess objects i.e. suffix the code with `.map {|p| AppStruct.new(*p)`:

```ruby
def parse_ps(ps_string)
  ps_string.scan(/([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+([^\s]+)\s+(.*)/).map { |p| AppProcess.new(*p) }
end
```